### ISSUE TITLE
Memory leak: consumption peak on column width set

### ISSUE THREAD

**Issue by SymbioticKilla** [EPPlusSoftware/EPPlus#576]
.NET 6.0
EPPlus 5.8.5(licensed)

Hi,

I have noticed strange behavior with memory consumption. Is it normal?
Thanks!

![image](https://user-images.githubusercontent.com/59652865/149569681-cfddb463-cc2c-49b7-9c41-46d9633278ee.png)

After this line:
`ws.Column(1).Width = 55.0;`

![image](https://user-images.githubusercontent.com/59652865/149569703-b31d658b-7345-4333-b8d3-48dea210c562.png)

After that it goes back to normal and on further excel exports does not happen.

**Comment by JanKallman** [comment]
Hi,
Setting the column Width will load and adjust the size of all drawings (charts, shapes, pictures etc). See https://github.com/EPPlusSoftware/EPPlus/blob/3410582e2ae4d129f03ce216a49b4c42c7fa54a6/src/EPPlus/ExcelColumn.cs#L146. 
Without knowing the content of your workbook I really can't say, but if you have for example pictures in the worksheet, EPPlus will load the images from the package at this line(if you haven't accessed the Drawings collection before).

**Comment by SymbioticKilla** [comment]
@JanKallman 
Hi,

thank you for reply. The problem is that there is no data at this point.
Simple Example:
![image](https://user-images.githubusercontent.com/59652865/149728632-36cb2018-6e03-4f6e-b52f-e5b0ac247024.png)

**Comment by JanKallman** [comment]
Ok, I understand, a blank workbook with no drawings then. I'll add a test to see what is going on.

**Comment by JanKallman** [comment]
Hello again,
This is normal. When setting the column width, the underlaying cell store is initialized and space for column and row indexes are allocated (for faster inserts and read access). Room for 32 columns, with 32 pages with 64 rows each are allocated to start with. Columns not used are set to null, so underlaying page/row are set the first time a column/value is set. In your case a column index for column 1 is created. The cells store memory is released when disposing the package.

**Comment by SymbioticKilla** [comment]
Thanks for clarification!
250MB+ is pretty much for simple Excel exports. Is there any option to configure these pre allocations?

**Comment by JanKallman** [comment]
No currently not. 250M sounds way to much though. The cell store index consists of int's and structs that should take very little resources. I'll have another look to see if something else is going on at some point.

**Comment by JanKallman** [comment]
EPPlus loads a static dictionary with font heights/widths used for measuring column widths that seems to cause this issue... https://github.com/EPPlusSoftware/EPPlus/blob/3410582e2ae4d129f03ce216a49b4c42c7fa54a6/src/EPPlus/FontSize.cs#L280
Quite a lot of data, but mine peeks at 50-80k when running the code in debug, so I'm unsure why you get as much as 250k. Anyway, not a good solution, so I'll see if I can find a better way of reading this data instead of loading it all into a dictionary.  Thanks for pointing this out.

**Comment by SymbioticKilla** [comment]
I have noticed it in productive env in alpine docker image and the same consumption I get on Windows with Visual Studio on debug. This peak is very short => after width set it goes down
![image](https://user-images.githubusercontent.com/59652865/149909393-b84cd63b-3337-4e68-9caf-a553eac9dadb.png)

**Comment by Polovykh** [comment]
@SymbioticKilla hello, just FYI, 

I've got a similar problem in process of accessing ExcelWorkbook.Worksheets property at the first time
![image](https://user-images.githubusercontent.com/6101013/150483412-3841e4ce-70b7-4701-8a5e-b0d129bd199d.png)

EPPlus 5.7.5: (_the arrow points to the moment when I asked an export_)
![image](https://user-images.githubusercontent.com/6101013/150483531-5a30a43f-3667-44ab-8a87-64bb66f4438b.png)

EPPlus 5.8.0 (_the arrow points to the moment when I asked an export_)
![image](https://user-images.githubusercontent.com/6101013/150483576-c38b9c47-efe8-42fd-831c-3e9e1632991f.png)

250M peak is may be ok for "classic" types of hosting, but it's too much for an usual microservice.

P.S. Screenshots above captured on my local Windows machine, if it matters. But the last time when I was trying to generate report inside the container based on "mcr.microsoft.com/dotnet/aspnet:6.0", I've got the OOM. =)
